### PR TITLE
Adding PyBDA: trial 2

### PR DESCRIPTION
**Pull request by dirmeier**
:information_source:
Bioconda has finished the [GCC7 migration](https://github.com/bioconda/bioconda-recipes/issues/13578). If you are dealing with C/C++ or Python package it can be that you need to rebuild other dependent packages. [Bioconda utils - update-pinning](https://bioconda.github.io/updating.html#updating-recipes-for-a-pinning-change) will assist you with that. If you have any questions please use [issue 13578](https://github.com/bioconda/bioconda-recipes/issues/13578).

----------------

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by dirmeier** [comment]
Ok, this seems to work now. Could someone from @bioconda/core please have a look at this? PyBDA is a package for big data analysis of biological data sets using Spark and Snakemake. Would be great if this would get merged. 

Thanks,
Simon

File: recipes/pybda/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 541908e3874550de3fdceb118279e7530180e1fe9caf7319fe0e29408227f788
+
+build:
+  number: 0
+  skip: true # [not py35 and not py37]
```
Comment (dpryan79) [inline recipes/pybda/meta.yaml]: Aren't you going for `skip: True  # [not py36]` ? You can probably just remove this, make it `noarch: python` and then specify `python 3.6` in the host and run sections.

File: recipes/pybda/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+requirements:
+  host:    
+    - python
```
Comment (dpryan79) [inline recipes/pybda/meta.yaml]: Given the run section, I presume you need `- python 3.6` here.

File: recipes/pybda/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  home: https://github.com/cbg-ethz/pybda
+  license: GNU General Public v3 (GPLv3)
+  license_family: GPL3
+  summary: Big biological data analytics powered by Apache Spark
+  doc_url: https://pybda.readthedocs.io/en/latest/
+  dev_url: https://github.com/cbg-ethz/pybda
+
+extra:
+  recipe-maintainers:
+    - dirmeier
```
Comment (dpryan79) [inline recipes/pybda/meta.yaml]: You can remove the `extra:` section, we don't end up using it in bioconda (not that it's hurting anything).

File: recipes/pybda/meta.yaml (hunk 1 above)
Comment (dpryan79) [reply recipes/pybda/meta.yaml]: This is actually causing it to not build anything, which is why all of the testing passes (there's nothing to fail).

Comment (dirmeier) [reply recipes/pybda/meta.yaml]: Oh man, thanks!

**Comment by dirmeier** [comment]
Right...I had the same kind of error with another PR and couldn't fix it. Any ideas @dpryan79? Thanks for your help!

**Comment by dpryan79** [comment]
@dirmeier It passes CI testing and builds a package now. If you post a link to the other PR I'll have a look.

**Comment by dirmeier** [comment]
Wow, thanks so much, @dpryan79 . The other PR was the same package. I closed it, since I assumed it failed due to some old the pinning or so. So no need to look at it. Thanks again!

**Comment by dpryan79** [comment]
Great, go ahead and merge this whenever you're ready then.

**Comment by dirmeier** [comment]
Cool, thanks for the help again!

**Pull request merged**